extensions/VisualEditor/wikia/modules/ve/ui/pages/ve.ui.WikiaParameterPage.js

```diff
@@ -17,9 +17,23 @@
  * @param {Object} [config] Configuration options
  */
 ve.ui.WikiaParameterPage = function VeUiWikiaParameterPage( parameter, name, config ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this function anonymous by removing its name: 'function() {...}'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ANamedFunctionExpression)